client/assets/stylesheets/_main.scss

```diff
@@ -259,9 +259,9 @@ th {
 }
 
 /* Links */
-a {
-	text-decoration: none;
-}
+// a {
```

Comment:
No doubt on your radar, but of course we'll just want to delete this outright, when we move out from draft state.

Reply:
Yes, of course! I'll go ahead and remove it.